common/src/main/java/com/wynntils/models/containers/type/LootChestTier.java

```diff
@@ -8,36 +8,40 @@
 import java.util.regex.Pattern;
 import net.minecraft.client.gui.screens.Screen;
 
-public enum LootChestType {
-    TIER_1(Texture.CHEST_T1, "Loot Chest 1", Pattern.compile("Loot Chest §7\\[§f✫§8✫✫✫§7\\]")),
-    TIER_2(Texture.CHEST_T2, "Loot Chest 2", Pattern.compile("Loot Chest §e\\[§6✫✫§8✫✫§e\\]")),
-    TIER_3(Texture.CHEST_T3, "Loot Chest 3", Pattern.compile("Loot Chest §5\\[§d✫✫✫§8✫§5\\]")),
-    TIER_4(Texture.CHEST_T4, "Loot Chest 4", Pattern.compile("Loot Chest §3\\[§b✫✫✫✫§3\\]"));
+public enum LootChestTier {
+    TIER_1(Texture.CHEST_T1, 1, Pattern.compile("Loot Chest §7\\[§f✫§8✫✫✫§7\\]")),
+    TIER_2(Texture.CHEST_T2, 2, Pattern.compile("Loot Chest §e\\[§6✫✫§8✫✫§e\\]")),
+    TIER_3(Texture.CHEST_T3, 3, Pattern.compile("Loot Chest §5\\[§d✫✫✫§8✫§5\\]")),
+    TIER_4(Texture.CHEST_T4, 4, Pattern.compile("Loot Chest §3\\[§b✫✫✫✫§3\\]"));
 
     private final Texture waypointTexture;
-    private final String waypointName;
+    private final int waypointTier;
     private final Pattern titlePattern;
 
-    LootChestType(Texture waypointTexture, String waypointName, Pattern titlePattern) {
+    LootChestTier(Texture waypointTexture, int waypointTier, Pattern titlePattern) {
         this.waypointTexture = waypointTexture;
-        this.waypointName = waypointName;
+        this.waypointTier = waypointTier;
         this.titlePattern = titlePattern;
     }
 
     public Texture getWaypointTexture() {
         return waypointTexture;
     }
 
+    public int getWaypointTier() {
+        return waypointTier;
+    }
+
     public String getWaypointName() {
-        return waypointName;
+        return "Loot Chest " + waypointTier;
```

Comment:
i18n? Or at least a i18n FIXME?

Reply:
Not really something to i18n IMO, it's more like internal knowledge of the wynn world, than something we represent to the user (this method is not even used atm).